src/condor_credd/condor_credmon_oauth/credmon/CredentialMonitors/LocalCredmon.py

```diff
@@ -120,6 +123,18 @@ def generate_access_token_info(self, username: str, token_name: str) -> TokenInf
         """
         # Create scopes from user and group templates
         scopes = [self.authz_template.format(username=username)]
+        if self.authz_template_expr:
+            try:
+                eval_context = classad.ClassAd({
+                    "Username": username,
+                    "Scopes": classad.ExprTree(self.authz_template_expr),
+                })
+                eval_scopes = eval_context.eval("Scopes")
+                if not isinstance(eval_scopes, str):
+                    raise TypeError(f"ClassAd expression '{self.authz_template_expr}' did not evaluate to a string")
+                scopes.append(eval_scopes)
+            except Exception:
+                self.log.exception(f"Not adding scopes from {self.credmon_name}_CREDMON_AUTHZ_TEMPLATE_EXPR due to exception")
```

Comment:
Can we log details of the exception here like we do here https://github.com/htcondor/htcondor/blob/efcf0672924efb3f7a715d759d9ceb51156a19e6/src/condor_credd/condor_credmon_oauth/credmon/CredentialMonitors/LocalCredmon.py#L183-L185?

Reply:
I could do that if I were catching specific errors... I'm a little hesitant to do that until I get some examples of how this goes wrong. `logging.exception()` will at least print the traceback alongside the error message, so there should already be useful information in the log from that.